src/Build/BackEnd/Shared/BuildRequestConfiguration.cs

```diff
@@ -547,6 +548,12 @@ internal int ResultsNodeId
             set => _resultsNodeId = value;
         }
 
+        public bool SkippedFromStaticGraphIsolationConstraints
```

Comment:
Affects non-isolation execution: new translatable bool that never gets set or read outside isolation execution